### ISSUE TITLE
🐛 [BUG] - Missing type in user data (when user is no longer accessing the workspace - TBC)

### ISSUE THREAD

**Issue by al1p** [ultimate-notion/ultimate-notion#39]
### Problem description

Testing v.0.5 
going back to the basic use cases, I get the following error in some case . 

```
from rich import inspect, print
import ultimate_notion as uno

def get_page(uuid: str):
    with uno.Session() as notion:
        page = notion.get_page(uuid)
    return page


def search_page(page_title: str):
    with uno.Session() as notion:
        page_res = notion.search_page(page_title)
        inspect(page_res)
        page = page_res.item()
    return page



if __name__ == "__main__":
    UUID2 = "88e76..f1" # WORKING
    UUID1 = "9e792...4fa6747c5a2" # NOT WORKING

    page = get_page(UUID2)
    searched_page = search_page(page.title)
    print(search_page)
```

```
Missing type in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'}. Most likely a User object without type
Missing type in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'}. Most likely a User object without type
Missing type in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'}. Most likely a User object without type
Traceback (most recent call last):
  File "/Volumes/SSDext1TB/Documents/GitRepo/PERSO/notion-automation/src/notion_automation/experiment_ultimate.py", line 24, in <module>
    getting_started()
  File "/Volumes/SSDext1TB/Documents/GitRepo/PERSO/notion-automation/src/notion_automation/experiment_ultimate.py", line 11, in getting_started
    page = notion.search_page(PAGE_TITLE).item()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/SSDext1TB/Documents/GitRepo/PERSO/notion-automation/.venv/lib/python3.12/site-packages/ultimate_notion/session.py", line 245, in search_page
    pages = [
            ^
  File "/Volumes/SSDext1TB/Documents/GitRepo/PERSO/notion-automation/.venv/lib/python3.12/site-packages/ultimate_notion/obj_api/iterator.py", line 135, in __call__
    api_list = ObjectList.model_validate(result_page)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/SSDext1TB/Documents/GitRepo/PERSO/notion-automation/.venv/lib/python3.12/site-packages/pydantic/main.py", line 568, in model_validate
    return cls.__pydantic_validator__.validate_python(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.ValidationError: 3 validation errors for ObjectList
results.65.properties.In charge.people.0
  Value error, Missing 'type' in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'} [type=value_error, input_value={'object': 'user', 'id': ...4f5d-9efd-57baf8210980'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
results.65.properties.Last edited by.last_edited_by
  Value error, Missing 'type' in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'} [type=value_error, input_value={'object': 'user', 'id': ...4f5d-9efd-57baf8210980'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
results.89.properties.Last edited by.last_edited_by
  Value error, Missing 'type' in data {'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'} [type=value_error, input_value={'object': 'user', 'id': ...4f5d-9efd-57baf8210980'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.8/v/value_error
```

I started debugging this, and something is preventing the model validation of
`obj_api.iterator.EndpointIterator.__call__()`:
```         api_list = ObjectList.model_validate(result_page)``
It may be the following property :

```
    'properties': {
      ...
        'In charge': {'id': 'Oz%5BI', 'type': 'people', 'people': [{'object': 'user', 'id': '8a9a7922-6f1d-4f5d-9efd-57baf8210980'}]},
...
```

It seems that this user is no longer in our list of users... 

### Expected behaviour

No crash ;-
or an easier  error message - I would be great if the trace could include the id / title of the page with the problem 😃 

### Steps to reproduce

create page
Add Property of type "person"
Assign to "User1"
remove "User1" from the list of shared users

This is "probably" the situation of the page I found.

### Version

0.5

### Installation

pip

### Python version

3.12

### Operating system

Mac

**Comment by al1p** [comment]
Found another case similar to the one above : same situation = property of type person where the user is no longer part of the workspace...

**Comment by FlorianWilhelm** [comment]
Hi @al1p and thank you. I merged the PR and will make a new release 0.5.1
